src/components/layout/tables/NameTable/styles.css

```diff
@@ -0,0 +1,33 @@
+.assets-table-header {
+  display: flex;
+  position: relative;
+  flex-direction: row;
+  height: 50px;
+  width: 100%;
+  border-radius: 6px;
+  background-color: #323232;
+  align-items: center;
+}
```

Comment:
can we not achieve these attributes with our existing CSS classes?


`flex-row` `center`, etc.